# File where major input definitions and values are provided
---
# Camera calibration info
camera_info:
  baseline: 0.12       # meters
  delta_d: 1.0         # stereo matcher disparity error in pixels
  h_fov: 85            # ZED's horizontal field of view in degrees
  cam_pitch: 10        # camera inclination in degrees
  cam_height: 1.0      # camera height in meters
  img_width: 1280      # pixels
  img_height: 720      # pixels
  fx: 692.95849609375  # pixels
  fy: 636.41424560546  # pixels
  cx: 692.95849609375  # pixels
  cy: 382.80633544921  # pixels

# Static camera pose information
camera_tf:
  pitch: 10   # degrees
  height: 1.2 # meters

# Particle filter constants to perform inovation and number of particles
# std_rot_a = alpha_1 * |dt_rot_a| + alpha_2 * |t|
# std_trans = alpha_3 * |dt_trans| + alpha_4 * |dt_rot_a + dt_rot_b|
# std_rot_b = alpha_1 * |dt_rot_b| + alpha_2 * |t|
pf:
  n_particles: 2000 # number of particles
  alpha_1: 0.5
  alpha_2: 0.5
  alpha_3: 0.5
  alpha_4: 0.5
...
